src/components/List/TESTS_ACTIVE/PercentageRow.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import {
+  Row,
+  Bold,
+  LocaleNumberWithPlus,
+  LocaleNumber,
+  Text,
+  LocaleNumberWithPercent,
+} from '../../shared/ui/New';
+
+function PercentageRow({ title = '', numerator, denominator, percent }) {
+  return (
+    <Row>
+      <Text> {title}: </Text>
+      <Bold>
+        <LocaleNumberWithPlus number={numerator} />
```

Comment:
Lower down I mention there's a native JS way to format these numbers. Formatting numbers and text should be the job of functions, not component.